tasks/configure.yml

```diff
@@ -48,8 +48,33 @@
     state: directory
     owner: "{{ pdns_rec_file_owner }}"
     group: "{{ pdns_rec_file_group }}"
-    mode: 0750
+    mode: "{{ pdns_rec_config_include_dir_mode }}"
   when: "pdns_rec_config['include-dir'] is defined"
+  register: _pdns_recursor_configuration_include_dir
+
+- block:
+
+  - name: Ensure that the PowerDNS Recursor configuration from-files directory exists
```

Comment:
wrong indentation

Reply:
The block indentation require some extra indentation of tasks inside it. 
https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_blocks.html#grouping-tasks-with-blocks